webcompat/static/css/src/grid.scss

```diff
@@ -0,0 +1,90 @@
+.grid {
+  margin-left: 1 / 2 * $unit-space;
```

Comment:
it would make sense to group those to functions, right? So we have a consistent unit-space and it does not seem random?

Reply:
Sugestions? 😄

Reply:
e.g. for half margin `1 / 2 * $unit-space;`  etc. Going through the code and checking, if those are repeating, would probably make sense?